crates/pixi_config/src/lib.rs

```diff
@@ -1276,6 +1276,12 @@ impl Config {
     /// The given config will have higher priority
     #[must_use]
     pub fn merge_config(mut self, mut other: Config) -> Self {
+        tracing::trace!(
```

Comment:
Do we still need to keep it? I think we could remove it, or if necessary, change to debug level